src/realm.js

```diff
@@ -258,7 +258,7 @@ export class Realm {
 
     this.start = Date.now();
     this.compatibility = opts.compatibility !== undefined ? opts.compatibility : "browser";
-    this.maxStackDepth = opts.maxStackDepth || 225;
+    this.remainingCalls = opts.maxStackDepth || 112;
```

Comment:
Had to reduce default maximum a bit to ensure that we don't run out of default node.js space before running out of default Prepack stack space.